### PR TITLE
Add contact form

### PR DESCRIPTION
**Pull request by Etsubdink-Demeke**
This pull request is created for the changes I made in my project including 
-Add a contact form to the index.html file 
-style the contact form according to the Figma design 
-configure formspre for the action attribute in the form tag

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -229,10 +229,24 @@ <h2 id="title">About Myself</h2>
...
+              <label for="email"></label>
+              <input type="email" name="email" placeholder="enter your email " required>
+            </li>
+            <li>
+              <label for="comment"></label>
+              <textarea id="comment" maxlength="500" name="comment" placeholder="leave your comment" required></textarea>
+            </li>
+            <li class="button">
+              <button type="submit">Get in touch</button>
+              </li>
```
Comment (ademibowale) [inline index.html]: - [ ] I would kindly suggest you to remove all unordered list tag from the form tag,becouse Forms very rarely consist of unordered lists, so using UL/LI is normally just as unsemantic as using TABLE.
LABEL for labels is definitely semantically correct, though.

Comment (ademibowale) [inline index.html]: - [ ] I would kindly suggest you to ensure all input fields are wrapped in a single `form` element for proper html semantic reasons.

**Review by ademibowale — CHANGES_REQUESTED** [review]
## STATUS: Changes Required :recycle:

### Hi @Etsubdink-Demeke , 👋 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



### To Highlight 🥇

✅ Descriptive Pull Request title and summary. ✔️
✅ Good use of Github flow ✔️
✅All linters are checked.
✅Project is professionCollect data by using the [Formspree service](https://formspree.io/html).
✅The right GitHub flow method has been used.
✅Professional pull request title and description.

### Changes Required

- [x]  _Please check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me on( @Adebowale ) in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by sebalp100 — APPROVED** [review]
Hi @Etsubdink-Demeke , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

thumbs-up-double-thumbs-up


## Highlights

✔️ Correct use of the Form element to wrap all the inputs
✔️ Removed unnecessary ul/li elements
✔️ Form works correctly

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**